demo/index.html

```diff
@@ -0,0 +1,84 @@
+<!--
+    @license
+    Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
+    This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+    The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+    The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+    Code distributed by Google as part of the polymer project is also
+    subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<!doctype html>
+<html>
+  <head>
+    <title>iron-range</title>
+    
+    <script src="../../webcomponentsjs/webcomponents.js"></script>
```

Comment:
As part of the transition, please change all references to `webcomponents.js` to `webcomponents-lite.js`, which is a package that omits ShadowDOM polyfills.